src/sage/doctest/rif_tol.py

```diff
@@ -25,12 +25,12 @@
 # ****************************************************************************
 
 from sage.doctest.marked_output import MarkedOutput
+from sage.rings.real_mpfi import RealIntervalField, RealIntervalFieldElement
 
+_RIFtol: 'RealIntervalField | None' = None
```

Comment:
I've never seen this Python syntax - what does it do?

Reply:
Yes, it's a type annotation!